eng/versioning/version_client_java_files.txt

```diff
@@ -10,7 +10,7 @@ sdk/keyvault/azure-security-keyvault-keys/src/main/java/com/azure/security/keyva
 sdk/keyvault/azure-security-keyvault-secrets/src/main/java/com/azure/security/keyvault/secrets/AzureKeyVaultConfiguration.java
 sdk/storage/azure-storage-blob/src/main/java/com/azure/storage/blob/implementation/util/BuilderHelper.java
 sdk/storage/azure-storage-blob-cryptography/src/main/java/com/azure/storage/blob/specialized/cryptography/BlobCryptographyConfiguration.java
-sdk/storage/azure-storage-file/src/main/java/com/azure/storage/file/BuilderHelper.java
-sdk/storage/azure-storage-file/src/main/java/com/azure/storage/file/FileConfiguration.java
+sdk/storage/azure-storage-file-share/src/main/java/com/azure/storage/file/BuilderHelper.java
+sdk/storage/azure-storage-file-share/src/main/java/com/azure/storage/file/FileConfiguration.java
```

Comment:
Both of these file paths are incorrect. The correct, current paths from the previous PR are below
sdk/storage/azure-storage-file-share/src/main/java/com/azure/storage/file/share/FileConfiguration.java
sdk/storage/azure-storage-file-share/src/main/java/com/azure/storage/file/share/implementation/util/BuilderHelper.java

Reply:
Done.